modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaClientCodegen.java

```diff
@@ -392,6 +392,10 @@ public void processOpts() {
             forceSerializationLibrary(SERIALIZATION_LIBRARY_JACKSON);
             supportingFiles.add(new SupportingFile("ParamExpander.mustache", invokerFolder, "ParamExpander.java"));
             supportingFiles.add(new SupportingFile("EncodingUtils.mustache", invokerFolder, "EncodingUtils.java"));
+            supportingFiles.add(new SupportingFile("auth/DefaultApi20Impl.mustache", authFolder, "DefaultApi20Impl.java"));
```

Comment:
@hugo-ma-alves quick question. How is the file `"DefaultApi20Impl` named?

Reply:
Just saw it: `import com.github.scribejava.core.builder.api.DefaultApi20;`

Please ignore my question.